modules/drivers/velodyne/params/VLP32C_calibration_example.yaml

```diff
@@ -0,0 +1,225 @@
+  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 53, max_intensity: 255,
+  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
+- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
+  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 54, max_intensity: 255,
+  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
+- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
+  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 55, max_intensity: 255,
+  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
+- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
+  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 56, max_intensity: 255,
+  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
+- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
+  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 57, max_intensity: 255,
+  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
+- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
+  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 58, max_intensity: 255,
+  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
+- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
+  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 59, max_intensity: 255,
+  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
+- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
+  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 60, max_intensity: 255,
+  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
+- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
+  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 61, max_intensity: 255,
+  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
+- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
+  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 62, max_intensity: 255,
+  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
+- {dist_correction: 0.0, dist_correction_x: 0.0, dist_correction_y: 0.0, focal_distance: 0.0,
+  focal_slope: 0.0, horiz_offset_correction: 0.0, laser_id: 63, max_intensity: 255,
+  min_intensity: 0, rot_correction: 0.0, vert_correction: 0.0, vert_offset_correction: 0.0}
+num_lasers: 64
```

Comment:
maybe 32 lasers here

Reply:
This is the official configuration file.

Reply:
> This is the official configuration file.

ok, 👍